multisrc/overrides/mangathemesia/sushiscan/src/SushiScan.kt

```diff
@@ -32,6 +35,9 @@ class SushiScan : MangaThemesia("Sushi-Scan", "https://sushiscan.net", "fr", man
         else -> SManga.UNKNOWN
     }
 
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request =
+        GET("$baseUrl/page/$page/?s=$query")
```

Comment:
This is error-prone to URL query parameter encoding issues. You can use OkHttp's HttpUrl builder instead.

```suggestion
    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
        val url = "$baseUrl/page/$page/".toHttpUrl().newBuilder()
            .addQueryParameter("s", query)
            .build()
            
        return GET(url, headers)
    }
```

Make sure to add the proper imports.

Reply:
OK, is this URL query parameter encoding issue stuff new?